Dockerfile

```diff
@@ -3,12 +3,16 @@
 #
 
 # Based on docker official node image
-FROM node:5.10.1
+FROM node:8
 
 # Install "nodemon" and "buster" globally
+RUN chown -R node:node /usr/local/lib/node_modules
+RUN chown -R node:node /usr/local/bin
+USER node
 RUN npm install -g nodemon buster
 
 # Setup workspace
+USER root
```

Comment:
Same here.

Reply:
Encountered some EACCESS error from npm without it, see https://docs.npmjs.com/getting-started/fixing-npm-permissions